recipe/meta.yaml

```diff
@@ -6,63 +6,65 @@ package:
   version: {{ version }}
 
 source:
-  fn: opencv-{{ version }}.tar.gz
-  url: https://github.com/opencv/opencv/archive/{{ version }}.tar.gz
-  sha256: f1b87684d75496a1054405ae3ee0b6573acaf3dad39eaf4f1d66fdd7e03dc852
+  - url: https://github.com/opencv/opencv/archive/{{ version }}.tar.gz
+    fn: opencv-{{ version }}.tar.gz
+    sha256: f1b87684d75496a1054405ae3ee0b6573acaf3dad39eaf4f1d66fdd7e03dc852
+    patches:  # [win]
+      - 01_win_fix_cmake_config_sufix.patch  # [win]
+  - url: https://github.com/opencv/opencv_contrib/archive/{{ version }}.tar.gz
+    fn: opencv_contrib-{{ version }}.tar.gz
+    sha256: 298c69ee006d7675e1ff9d371ba8b0d9e7e88374bb7ba0f9d0789851d352ec6e
+    folder: opencv_contrib-{{ version }}
 
 build:
-  number: 200
+  number: 204
   # Python2.7 support dropped: https://github.com/opencv/opencv/issues/8481
   skip: true  # [win and py27]
-  features:                    # [not win]
+  features:  # [not win]
     - blas_{{ blas_variant }}  # [not win]
 
 requirements:
   build:
     - python
     - toolchain
```

Comment:
Would be good to update to the new compiler syntax and split `build`/`host`.

ref: https://conda-forge.org/docs/meta.html#compilers
ref: https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#host

Reply:
Hello, I'll need help here too.
1- The compiler expected would by compiler "{{ compiler('python_c') }}
" as described in https://conda.io/docs/user-guide/tasks/build-packages/compiler-tools.html ? or where shall I get the compiler name from ?

2- Host and Run concepts, are

Reply:
2 - Host and Run sections are not very clear to me. Host sections refer to packages available/belonging to the host that are not packaged with the crafted package? Run dependencies are installed from conda available channels when matching the version .. ?

Reply:
So PR ( https://github.com/conda-forge/opencv-feedstock/pull/113 ) takes care of a lot of this. Will try to get that in first and then can circle back.

Reply:
ok, great.